lib/middleware/validate-request-query-middleware.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+/**
+ * Parse an incoming request and validate that the query matches a schema.
+ *
+ * Use this as middleware in your route files.
+ */
```

Comment:
Documentation was put above the `"use strict"` in lib/middleware/read-body-buffer-middleware.js. Do we have a convention regarding the placement of documentation vs `"use strict"` at the top of files?